graph-builder/src/graph.rs

```diff
@@ -31,6 +31,11 @@ lazy_static! {
         "Number of releases in the final graph, after processing"
     )
     .unwrap();
+    static ref GRAPH_LAST_SUCCESSFUL_REFRESH: IntGauge = IntGauge::new(
+        "graph_last_successful_refresh_timestamp",
```

Comment:
maybe `graph_upstream_last_successful_refresh_timestamp` for consistency?

Reply:
I'd say no, at this point this metrics is not anymore related to reaching upstream (e.g. a plugin could be failing, without any quay-fetching issues) and it is already consistent with the other `graph_final_releases` one.